src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -192,12 +192,11 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
   if (changeRequest.accepted!) actionDropdown = implementCrDropdown;
 
   return (
-    <>
-      <PageTitle
-        title={`Change Request #${changeRequest.crId}`}
-        previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]}
-        actionButton={actionDropdown}
-      />
+    <PageLayout
+      title={`Change Request #${changeRequest.crId}`}
+      previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]}
+      actionButton={actionDropdown}
+    >
```

Comment:
<img width="823" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/c9513ea4-79d2-43c3-be58-c1d741ff6035">
same here, the button moves down weirdly soon when you shrink the screen